plugins/com.google.cloud.tools.eclipse.appengine.newproject/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Cloud Tools for Eclipse App Engine New Project Generation
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.appengine.newproject;singleton:=true
+Bundle-Version: 0.1.0.qualifier
+Bundle-Vendor: Google, Inc.
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Require-Bundle: org.eclipse.jface.viewers,
+  org.eclipse.jface.wizard,
+  org.eclipse.ui,
+  javax.servlet;bundle-version="2.5.0";visibility:=reexport,
```

Comment:
And please don't re-export `javax.servlet` and `javax.servlet.jsp`: it's an anti-pattern.  Eclipse has several bundles that does this for historical reasons; we'd like to remove them but it causes API breakage.